### ISSUE TITLE
Add MAGETWO-52799 as a known issue to the release notes

### ISSUE THREAD

**Issue by korostii** [magento/devdocs#1120]
Feedback on page: /guides/v2.1/release-notes/ReleaseNotes2.1.6CE.html
There is a line saying "The Web Setup Wizard now works when Magento is installed in /pub. (GITHUB-4159)"

However, the following comments from Magento contributors clearly state the opposite:
https://github.com/magento/magento2/issues/4159#issuecomment-232372120
https://github.com/magento/magento2/issues/5247#issuecomment-295247765

[The issue](https://github.com/magento/magento2/issues/4159) [has](https://github.com/magento/magento2/issues/5235) [been](https://github.com/magento/magento2/issues/5247) [reported](https://github.com/magento/magento2/issues/7616) [numerous](https://github.com/magento/magento2/issues/7623) [times](https://github.com/magento/magento2/issues/8629) ever since the release of 2.1.0 (because, in fact, it still isn't fixed yet in 2.1.x branch)

Please move the info about MAGETWO-52799 in the release notes from "Fixed issues" to "Known issues" in order to warn the merchants and integrators about this issue. IMHO that would also somewhat help to prevent more and more new GitHub issues reported about it (not as much as a fast patch, but still).

**Comment by jfrontain** [comment]
hi @korostii -- I'm working on updating these notes right now. Thanks for this information!

**Comment by jfrontain** [comment]
@korostii, I'm sorry for the long delay in resolving this issue. Looks as though we removed the fixed issue description as you suggested, but did not describe the problem as a known issue. That MAGETWO issue is still unresolved, but we did not get the go-ahead to call it out in the release notes. I'm closing this particular doc issue and suggest that if you still feel strongly about the unresolved tech issue, you pursue it on the magento2 forum or even as a potential community issue. Thanks for both your patience and help!
